designsafe/celery.py

```diff
@@ -27,10 +27,10 @@
             'task': 'designsafe.apps.search.tasks.update_search_index',
             'schedule': crontab(minute=0, hour=0),
         },
-        'reindex_projects': {
-            'task': 'designsafe.apps.api.tasks.reindex_projects',
-            'schedule': crontab(hour="*/24", minute=0)
-        }
+       # 'reindex_projects': {
```

Comment:
is this intended?

Reply:
It is for now, the indexer breaks on some newer FR projects. Once this gets merged I'll tackle https://jira.tacc.utexas.edu/browse/DES-1587 and re-enable it.